api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -2962,7 +2962,7 @@ private void enableLabs() throws UnsupportedEncodingException {
                 "owa/labworkflow/index.html",
                   PihEmrConfigConstants.PRIVILEGE_APP_LABS,
                 null),
-                null));
```

Comment:
We now can control where the "Labs" component shows up via this new location tag.